utils/common_validation_test.go

```diff
@@ -24,53 +24,141 @@ func prepareForTest() (*http.Request, *httptest.ResponseRecorder) {
 
 func TestUnitHandleRequestValidation(t *testing.T) {
 	helperService := NewHelperService()
+
 	Convey("Fails validation when transaction ID does exists", t, func() {
 		var req, res = prepareForTest()
-		_, actual, _ := helperService.HandleTransactionIdExistsValidation(res, req, "")
+		transactionId_, valid, httpStatusCode := helperService.HandleTransactionIdExistsValidation(res, req, "")
```

Comment:
any reason why this is named like this.I thought transactionId should suffice

Reply:
done